packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.test.tsx

```diff
@@ -1039,6 +1039,36 @@ describe('ContextualMenu', () => {
       button.current!.openMenu();
       expect(document.querySelector('.SubMenuClass')).toEqual(null);
     });
+
+    it('Menu should correctly return focus to previously focused element when dismissed', () => {
+      const temp = ReactTestUtils.renderIntoDocument<HTMLDivElement>(
+        <div>
```

Comment:
nit: can remove <div> wrapper

Reply:
I cannot, at least not if I still want to get Temp and use it as an HTMLElement. At least that's how the typings seem to work out.